app/controllers/admin/domain_allows_controller.rb

```diff
@@ -28,6 +32,33 @@ def destroy
     redirect_to admin_instances_path, notice: I18n.t('admin.domain_allows.destroyed_msg')
   end
 
+  def export
+    authorize :domain_allow, :create?
+    csv = CSV.generate do |content|
```

Comment:
Probably makes sense to use headers here?

Reply:
I initially was generating headers, but I couldn't get them to be reliably detected when importing, so I ditched them again 😐